sources/packages/api/src/services/institution-location/institution-location.service.ts

```diff
@@ -51,4 +71,14 @@ export class InstitutionLocationService extends RecordDataModelService<Instituti
       .where('institution.id = :Id', { Id: institution_id})
       .getMany();
   }
+
+  async getInstitutionLocation(institution_id: number, location_id: number): Promise<InstitutionLocationsDetailsDto> {
+    return this.repo
+      .createQueryBuilder("institution_location")
+      .select(['institution_location.name', 'institution_location.data', 'institution.institutionPrimaryContact', 'institution_location.id'])
+      .leftJoin("institution_location.institution", "institution")
+      .where('institution.id = :Id and institution_Location.id = :locationId', { Id: institution_id, locationId: location_id})
```

Comment:
Please try to keep the consistency while declaring the variables: for instance "Id" vs "id".
I would recommend that line 80 be adjusted to: 
```
{ id: institutionId, locationId: locationId})
```